core/packages/test/config/launch-config.toml

```
@@ -41,27 +41,67 @@ default_args = [
     name = "rococo-dave"
     validator = true
 
-# Snowbridge
+## Bridge Hub
 [[parachains]]
+id = 1013
```

Should use the same ID as BridgeHub on Rococo, to minimize configuration overhead.

```suggestion
id = 1002
```

---

Do we not want to mimic the real-life parachain id's?

---

Rococo uses `1013` for BridgeHub. All other environments use `1002`. I am going to leave this as 1013.
```
Polkadot -> Statemint(1000), BridgeHub(Not deployed)
Kusama -> Statemine(1000), BridgeHub(1002)
Rococo -> Rockmine(1000), BridgeHub(1013)
Westend -> Westmint(1000), BridgeHub(1002)
```